ndm-daemonset.Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019-2020 The OpenEBS Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This Dockerfile builds node-disk-manager
+#
+FROM golang:1.14 as build
+
+ARG TARGETPLATFORM
+
+ENV GO111MODULE=off \
```

Comment:
With NDM now using go modules. we should keep `GO111MODULE=on`.   Currently it works because the dependencies are also stored in the tree in vendor folder.